### ISSUE TITLE
About pretrained model accuracy on LFW

### ISSUE THREAD

**Issue by zyt1378** [davidsandberg/facenet#814]
The accuracy for  the model 20180402-114759 is about 0.99550,my parameter setting is as follows:

detection and alignment:
python align_dataset_mtcnn.py \
/media/zheng/02FCF89DFCF88BE31/face_dataset/LFW/lfw \
/media/zheng/02FCF89DFCF88BE31/face_dataset/LFW/aligned_lfw_tf \
--image_size 160 \
--margin 32 \
--random_order \
--gpu_memory_fraction 0.25

embedding extraction:
python facenet_tf_extractor.py \
/media/zheng/02FCF89DFCF88BE31/face_dataset/LFW/aligned_lfw_tf \
pretrained_models/20180402-114759 \
--lfw_batch_size 64 \
--image_size 160 \
--lfw_pairs ../../pairs.txt \
--use_flipped_images \
--subtract_mean \
--use_fixed_image_standardization

validate result:
Accuracy: 0.99550+-0.00342
Validation rate: 0.98600+-0.00952 @ FAR=0.00100
Area Under Curve (AUC): 1.000
Equal Error Rate (EER): 0.004

why can't I reproduce the same result, my tensorflow version is 1.5,is this the problem?

**Comment by phoebushe** [comment]
my result is only:
Accuracy: 0.97750+-0.00396
Validation rate: 0.82100+-0.03709 @ FAR=0.00100
Area Under Curve (AUC): 0.997
Equal Error Rate (EER): 0.022

what's wrong?

**Comment by bing1zhi2** [comment]
my result is :
Accuracy: 0.98467+-0.00407
Validation rate: 0.90567+-0.01995 @ FAR=0.00067
Area Under Curve (AUC): 0.998
Equal Error Rate (EER): 0.015

Total number of images: 13233

@zyt1378  Do you konw why? Thx.

**Comment by bing1zhi2** [comment]
The result was good when I  add paramters  use_fixed_image_standardization and subtract_mean. I soon found this, but forgot to reply.

Accuracy: 0.99500+-0.00350
Validation rate: 0.97833+-0.01138 @ FAR=0.00067
Area Under Curve (AUC): 1.000
Equal Error Rate (EER): 0.006
